authorization/src/lib.rs

```diff
@@ -46,12 +44,11 @@ impl NonceAuth for WrappedAuth {
     }
 }
 
-pub struct AuthContract;
+pub struct ExampleContract;
 
-#[cfg_attr(feature = "export", contractimpl)]
-#[cfg_attr(not(feature = "export"), contractimpl(export = false))]
```

Comment:
Adding in these features isn't particularly important for this example as this example doesn't deal with how to import this contract into other contracts. Let's leave it out.